### ISSUE TITLE
Email settings refer to reblogging instead of boosting

### ISSUE THREAD

**Issue by kimroen** [tootsuite/mastodon#783]
Reblogging [was renamed to "Boosting"](https://github.com/tootsuite/mastodon/commit/8a4913fde0715f6fd6dafb3127c0017eaeb6f302), but it is still referred to as reblogging on [the preferences page](https://mastodon.social/settings/preferences):

![reblogs](https://cloud.githubusercontent.com/assets/520420/24617379/fe8a5884-1893-11e7-82c6-ed5cb958712c.png)

I'd think changing this line should do the trick ([`config/locales/simple_form.en.yml:41`](https://github.com/tootsuite/mastodon/blob/663e5378c08b7145a9fdeafc5ff0818a02cc3b63/config/locales/simple_form.en.yml#L41)):

```diff
-        reblog: Send e-mail when someone reblogs your status
+        reblog: Send e-mail when someone boosts your status
```

**Comment by azillion** [comment]
#806 builds on #805

**Comment by azillion** [comment]
Backend still has a lot of references to reblog though. 
https://github.com/tootsuite/mastodon/search?utf8=%E2%9C%93&q=reblog&type=

**Comment by kimroen** [comment]
> Backend still has a lot of references to reblog though.

Yes, but changing that would require a lot of migrations and similar risky changes, so I think that timing-wise it's probably good to focus on just the translations for now.

**Comment by kimroen** [comment]
Good catch on changing "status" to "toot" there as well, I didn't even notice that.

**Comment by kimroen** [comment]
This was solved by #805 - let's handle the "status" -> "toot" change separately 👍
